### DISCUSSION TITLE
DualLineTrails & Custom Width

### DISCUSSION THREAD

**Discussion by ayylmaoRotE** [Phobos-developers/Phobos#871] [Ideas and feedback]
Allow dual line trails on art.ini entries, e.g. aircraft where the center is still drawn at the center of the voxel/shape but the linetrails can be customised to a specific lateral offset, like FLH works:
```ini
[VXL]
UseLineTrail=yes
LineTrailColor=216,216,255  ; SJM
LineTrailColorDecrement=9  ; SJM
; New tags
DualLineTrail=yes
DualLineTrail.Offset=100
```
Custom linetrail thickness allows for a SAGE engine look to linetrails, e.g. on a rocket. Line trail width would be based on value:
```ini
[VXL]
UseLineTrail=yes
LineTrailColor=216,216,255  ; SJM
LineTrailColorDecrement=9  ; SJM
; New tag
LineTrail.Thickness=3
```
This would draw the 2 line trails at the end of both wings, rather than the center.

**Comment by MigEater** [comment]
I think it would be better if you were able to customise multiple line trails instead of just mirroring one, 
something like LineTrailX=yes where X is a 1-9 number etc, along with;
 
LineTrailX.Color=
LineTrailX.Decrement=
LineTrailX.Position= (use a FLH style positing instead of just the offset of 1 axis)
LineTrailX.Thickness=
